charts/virtual-nodes/values.yaml

```diff
@@ -2,12 +2,12 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-nodes: {}
-# - type: dgxa100.40g
-#   count: 2
-# - type: dgxa100.80g
-#   count: 2
-# - type: dgxh100.80g
-#   count: 2
-# - type: cpu.x86
-#   count: 2
+nodes: 
```

Comment:
The "values.yaml" is intentionally empty. We are providing examples of how to configure the nodes in values-example.yaml

Reply:
Then how about using `values-example.yaml` in the following command?  A user likes a simple copy & paste.
`helm install virtual-nodes charts/virtual-nodes -f charts/virtual-nodes/values-example.yaml`

Reply:
sure, that will work

Reply:
Updated it.